python/mxnet/ndarray/ndarray.py

```diff
@@ -673,6 +686,100 @@ def reshape(self, shape):
                                          ctypes.byref(handle)))
         return NDArray(handle=handle, writable=self.writable)
 
+    def flatten(self, **kwargs):
```

Comment:
Do we need to support positional arguments as well? At least, the following two usages of the op through `mx.nd` both work:
```python
mx.nd.tile(a, reps=(2, 3))
mx.nd.tile(a, (2, 3))
```

Reply:
Will add. Thanks